activerecord/lib/active_record/connection_adapters/abstract/schema_creation.rb

```diff
@@ -18,6 +18,9 @@ def visit_AddColumn(o)
           "ADD #{accept(o)}"
         end
 
+        delegate :quote_column_name, :quote_table_name, :quote_default_expression, :type_to_sql, to: :@conn
```

Comment:
Yes, still need, a method that has been defined by `delegate` will be public, regardless of the visibility of the current scope.